ReportSimulationOutput/measure.rb

```diff
@@ -413,6 +427,16 @@ def run(runner, user_arguments)
     end
 
     @timestamps = get_timestamps(timeseries_frequency)
+    if timeseries_frequency != 'none'
+      if add_timeseries_dst_column.is_initialized
+        @timestamps_dst = 'DST'
+        timestamps_dst = get_timestamps(timeseries_frequency, @timestamps_dst) if add_timeseries_dst_column.get
+      end
+      if add_timeseries_utc_column.is_initialized
+        @timestamps_utc = 'UTC'
```

Comment:
Do you really need the `@timestamps_dst` and `@timestamps_utc` global variables? They seem entirely unnecessary.